pymonetdb/filetransfer/__init__.py

```diff
@@ -153,7 +154,3 @@ def handle_download(self, download: "pymonetdb.filetransfer.downloads.Download",
         OVERWRITE ARBITRARY FILES on your computer
         """
         pass
-
-
-# Only import this at the end to avoid circular imports
```

Comment:
Removing this breaks the documentation (make doc)

Reply:
ah we should add that as a test then